config/locales/time/es.yml

```diff
@@ -2,6 +2,7 @@
 es:
   date:
     day_names:
+      -  # empty item to have correct weekday offset for %A
       - lunes
       - martes
       - miércoles
```

Comment:
A question outside of scope for your issue- We capitalize Monday in English but not Spanish and French. (day of week, months) Should day names and months be capitalized in Spanish and French?

Reply:
i wondered the same thing, we should ask design. I'll ask design in slack

Reply:
Day and month names are not capitalized in French or Spanish (unless they're at the beginning of a sentence)